client/src/pages/Profile/Profile.tsx

```diff
@@ -0,0 +1,78 @@
+import { useState, useEffect, useContext } from 'react';
+import { RouteComponentProps, Link } from 'react-router-dom';
+import { Card, CardMedia, Box, Avatar, Typography, Button } from '@material-ui/core';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import { fetchProfile } from '../../helpers/APICalls/fetchProfile';
+import { getContestsByUsername } from '../../helpers/APICalls/contest';
+import { User } from '../../interface/User';
+import { ContestAPIData } from '../../interface/Contest';
+import { AuthContext } from '../../context/useAuthContext';
+import useStyles from './useStyles';
+import ProfileTabs from '../../components/Profile Tabs/ProfileTabs';
+
+const Profile = ({ match }: RouteComponentProps): JSX.Element => {
+  const classes = useStyles();
+  const [profile, setProfile] = useState<User>();
+  const [contests, setContests] = useState<ContestAPIData[]>([]);
+  const { loggedInUser } = useContext(AuthContext);
+
+  useEffect(() => {
+    const { username } = match.params as { username: string };
+    const getProfile = async (username: string) => {
+      const response = await fetchProfile(username);
+      if (response) {
+        setProfile(response);
+      }
+    };
+    const getContestList = async (username: string) => {
```

Comment:
We have a separate context for contests, maybe consider using that as opposed to fetching all the contests once again?

Reply:
Hey Jeri, you make a good point. Thanks!